kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -148,6 +149,23 @@ trait ProcBuilder {
     proc
   }
 
+  val YARN_APP_NAME_REGEX: Regex = "application_\\d+_\\d+".r
+
+  def killApplication(line: String = lastRowOfLog): Int =
```

Comment:
can we return msg here directly and move following code into this method ?
```
 val killMessage = builder.killApplication() match {
            case id if id != 0 => "And failed to kill Application, please kill it manually. "
            case _ => "And killed Application successfully. "
          }
```

Reply:
Done